packages/utils/src/conversions.ts

```diff
@@ -0,0 +1,202 @@
+/**
+ * @module Conversions
+ * This module provides a collection of utility functions for converting
+ * between different numerical formats, particularly focusing on
+ * conversions involving bigints, hexadecimals and buffers.
+ * The module is structured with clear function naming to indicate
+ * the conversion direction (e.g., `bigIntToHexadecimal` for BigInt
+ * to hexadecimal, `bufferToBigint` for buffer to bigint) and employs
+ * type checks to ensure the correct handling of various input types.
+ * It also includes variations for both big-endian (`be`) and little-endian
+ * (`le`) conversions. It is important to note that when there is no prefix,
+ * the order of bytes is always big-endian.
+ */
+
+import { BigNumberish } from "./types"
+import { isHexadecimal, isStringifiedBigint } from "./number-checks"
+
+/**
+ * Converts a bigint to a hexadecimal string ensuring even length.
```

Comment:
If the function name makes byte-order explicit, the documentation should say what the byte order is, in order to describe what "be" stands for.  E.g. "Converts a bigint to a hexadecimal string ensuring even length.  Uses big-endian byte order."

This same comment applies to all of the functions in this file, so I'm not going to repeat it.  They should all make byte order explicit in the documentaiton.

Reply:
Right! I've added an explanation in the module description but it would be better to be explicit in those functions too 👍🏽